frontend/src/pages/projects/screens/projects/ProjectListView.tsx

```diff
@@ -109,26 +108,27 @@ const ProjectListView: React.FC<ProjectListViewProps> = ({ allowCreate }) => {
       />
       <ManageProjectModal
         open={!!editData}
-        onClose={() => {
+        onClose={(newProjectName) => {
```

Comment:
this doesnt get called for some reason and the modal doesnt get closed on cancel of the modal

Reply:
No, cause the modal is opened in `NewProject` when opened in the Project pages.